core/cas-server-core/build.gradle

```diff
@@ -18,6 +18,8 @@ dependencies {
     implementation project(":core:cas-server-core-services-api")
     implementation project(":core:cas-server-core-services-authentication")
 
+    testImplementation libraries.pac4jcore
```

Comment:
This cannot exist here. The module here is a "core" module and pac4j belongs to the support category.